stock_picking_batch_back2draft/readme/USAGE.rst

```diff
@@ -0,0 +1 @@
+When a picking batch is cancelled a button appears to return it back to draft.
```

Comment:
You should explain that this will reset picking linked to the batch that are backed to draft.

Reply:
Done